bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -277,6 +290,17 @@ public String getRoleAttribute() {
 
     @Override
     public String getAction() {
+        List<String> supportedSubmitActions = ComponentUtils.getSupportedSubmitActions(clientBuilderFactory);
+        String resourceType = resource.getValueMap().get("sling:resourceType", String.class);
+        supportedSubmitActions = Arrays.asList("spreadsheet");
```

Comment:
This code is overwriting the network call, shouldn't it merge ?

Reply:
this i will remove after testing and integrating with the submit service api. I have mentioned that url and this api can be changed during integration

Reply:
Once its merged in FAR, you can't change it

Reply:
Integrated with the live api and made the necessary changes in code and test mocks